docs/combat-mechanics/shielding.md

```diff
@@ -0,0 +1,154 @@
+---
+description: An explanation on shields, and how shield values are calculated.
+---
+
+# Shields
+
+:::info
+This document is mostly complete, but is missing a few notable exceptions. The concepts and mechanics outlined in this document are generally accurate.
+:::
+
+## Introduction
+
+Shields mitigate damage received from enemies in place of HP.
+
+The sections below detail the behaviour of how shields work in HSR.
+
+
+
+## Shield Absorption
+How do shields actually absorb damage?
+
+Simply this: Each shield has a Shield Effect Value. This Shield Effect Value (SEV) is equal to the number that appears when you cast a shield buff on a character. This SEV acts in place of HP when taking damage. Any damage taken is calculated before applied to a shield as per normal damage received calculation.
+
+![](/img/shields/shields1.jpg)
+
+In this instance, using March 7th’s Skill, Dan Heng receives a shield buff with 1271 SEV. This is reflected in character details as such.
+
+![](/img/shields/shields2.jpg)
+
+When damaged, the shield will absorb whatever damage is dealt to Dan Heng and apply it to the shield’s SEV instead.
+
+![](/img/shields/shields3.jpg)
+
+In this instance, the shield buff’s SEV takes 222 points of damage, reducing it to 1049, while Dan Heng takes no damage to HP. Notice that the Burn effect is still applied. This burn damage will be taken at the start of Dan Heng’s turn and applied against SEV rather than HP.
+
+![](/img/shields/shields4.jpg)
+
+Any overflow from an attack will be taken to HP as per normal.
+
+
+
+## SEV Calculation
+
+The formula to determine SEV applied by any particular shield buff is as follows
+
+$$
+SEV=(\% Scale\cdot Attribute+Extra\; SEV)\cdot\% Bonus
+$$
+
+Where $\% Scale$ is the % number that is used in calculation;
+
+$Attribute$ is the relevant stat that $\% Scale$ works based off;
+
+$Extra\; SEV$ is any extra SEV on skill cast; and
+
+$\% Bonus$ is the **additive** sum of all relic bonuses or miscellaneous shield bonuses.
+
+
+
+### Example Calculation
+In our example calculation, we use March 7th's Skill at Level 3 to apply the buff. March's skill at Level 3 is as follows:
+
+![](/img/shields/shieldscalc1.jpg)
+
+There is a quirk in Talent and Skill scaling. The numbers shown here are decimal values, but are rounded to integers. For more information on this behaviour, see Talents and Traces in the SRL.
```

Comment:
rounded -> truncated   (to indicate rounding down behaviour)

Maybe put a link to March's skill page https://srl.keqingmains.com/characters/ice/march_7th?ability=skill#abilities and point to use "[Show full table] button"